usermods/QuinLED_Dig_Uno_Temp_MQTT_XML/usermod.cpp

```diff
@@ -24,14 +25,29 @@ void userConnected()
 
 }
 
+//loop. You can use "if (WLED_CONNECTED)" to check for successful connection
 void userLoop()
 {
   temptimer = millis();
   
 // Timer to publishe new temperature every 60 seconds
   if (temptimer - lastMeasure > 60000) {
     lastMeasure = temptimer;
-    
+
+//Check for successful connection
+    if (WLED_CONNECTED){
+      sensor.requestTemperatures();
+
+//Gets prefered temperature scale based on selection in definitions section
+      #ifdef WLED_ENABLE_USERMODXML
```

Comment:
What if you have two or more usermods that are able to use this feature, both writing to e.g. usermodxml1?
I don't see locking mechanism.
IMO best way to interact with usermod is to call methods from usermod itself and not a usermod manupulating global variables.

Reply:
I dont how to do this. Im a beginner. Maybe close pr

Reply:
Another way could be to add/create a usermod API so usermods could exchange information.
In a similar way as addToJsonInfo() etc. is done.
The interaction between usermods was already implemented in Auto Save and Four Line Display usermods.

Reply:
Well, maybe we better close the PR. I have absolutely no clue to add this